native-sql-engine/core/src/main/scala/com/intel/oap/GazellePluginConfig.scala

```diff
@@ -83,6 +83,11 @@ class GazellePluginConfig(conf: SQLConf) extends Logging {
   val enableColumnarShuffledHashJoin: Boolean =
     conf.getConfString("spark.oap.sql.columnar.shuffledhashjoin", "true").toBoolean && enableCpu
 
+  // enable or disable fallback shuffle manager
+  val enableFallbackShuffle: Boolean = conf
```

Comment:
can you please also add a short note on how to use this feature?  and also make this default to false

Reply:
Added that in the description dialog